src/containers/ArtDetail.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import ArtworkDetail from '../components/artworks/ArtworkDetail';
+import { fetchByArtwork } from '../services/articAPI';
+
+export default class ArtDetail extends Component {
+    state = {
+      artwork: null,
+      loading: true,
+    };
+
+    async componentDidMount() {
+      
+      const { match } = this.props;
+      
+      const character = await fetchByArtwork(match.params.id);
+      this.setState({ character, loading: false });
```

Comment:
you don't have a piece of state called character. (I think you wanted artwork)....maybe ({ artwork: character}}